pkg/util/time.go

```diff
@@ -48,6 +48,20 @@ func ParseTime(s string) (int64, error) {
 	return 0, httpgrpc.Errorf(http.StatusBadRequest, "cannot parse %q to a valid timestamp", s)
 }
 
+func ParseDurationMs(s string) (int64, error) {
```

Comment:
We have this code in https://github.com/cortexproject/cortex/blob/master/pkg/querier/tripperware/queryrange/query_range.go#L382. Can you move it to this package and refactor it to use this new function?

Reply:
Move to utils package.